form/dynamic_form_modification.rst

```diff
@@ -591,6 +591,11 @@ callbacks only because in two different scenarios, the data that you can use is
 available in different events. Other than that, the listeners always perform
 exactly the same things on a given form.
 
+.. tip::
+
+    The ``FormEvents::POST_SUBMIT`` event does not allow to modify the form
+    the listener is bound to, but it allows to modify its parent.
+
```

Comment:
I think we can add it here. It supports this example https://symfony.com/doc/4.0/form/dynamic_form_modification.html#form-events-submitted-data.

Reply:
I'm not sure I understand your comment, what can we add here exactly?

Reply:
I just agree with your proposal! Let's add what you suggest :)